pkg/entrypoint/config/config.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type Common struct {
```

Comment:
all of this feels clumsy and overengineered, but I don't have a better suggestion right now.  I think we can check it in and make it nicer afterwards.  Examples: I think cmd/azure should pass a `*log.Entry` into the subcommands; I would like fewer .go files in each pkg/entrypoint/*, etc.

Reply:
problem with that is that loglevel is rootCmd level flag, so it can be read populated only when cobra is being initiated. And this happens only later in the process. an alternative is to parse loglevel using standard methods `flags`